<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1325</width>
    <height>771</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLineEdit" name="find_edit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>50</y>
     <width>431</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="top">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>0</y>
     <width>701</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(69, 167, 77);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="explain">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>220</y>
     <width>431</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="note">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>260</y>
     <width>431</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="show_lab">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>45</y>
     <width>571</width>
     <height>461</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>16</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;微软雅黑&quot;;
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="explain_lab">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>230</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>解释</string>
   </property>
  </widget>
  <widget class="QLabel" name="note_lab">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>270</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>笔记</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="edit_Box">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>470</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="ui.qrc">
     <normaloff>:/ui/edit.png</normaloff>:/ui/edit.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="find_btn">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>51</y>
     <width>129</width>
     <height>59</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(157, 38, 111);
font: 75 12pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>查找</string>
   </property>
   <property name="icon">
    <iconset resource="ui.qrc">
     <normaloff>:/ui/find.png</normaloff>:/ui/find.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>44</width>
     <height>44</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="set_w" native="true">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>571</width>
     <height>461</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
</string>
   </property>
   <widget class="QLabel" name="set_ico">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/ui/set.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="en_ico">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/ui/en.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>531</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QCheckBox" name="find_check">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt &quot;微软雅黑&quot;;
color: rgb(173, 64, 149);</string>
     </property>
     <property name="text">
      <string>强力搜索</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>65</y>
      <width>531</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="check_color1">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt &quot;微软雅黑&quot;;
color: rgb(173, 64, 149);</string>
     </property>
     <property name="text">
      <string>粉</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="check_color2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt &quot;微软雅黑&quot;;
color: rgb(173, 64, 149);</string>
     </property>
     <property name="text">
      <string>蓝</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="check_color3">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt &quot;微软雅黑&quot;;
color: rgb(173, 64, 149);</string>
     </property>
     <property name="text">
      <string>黑</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="check_color4">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>10</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt &quot;微软雅黑&quot;;
color: rgb(173, 64, 149);</string>
     </property>
     <property name="text">
      <string>绿</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>531</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QCheckBox" name="cet4_check">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt &quot;微软雅黑&quot;;
color: rgb(173, 64, 149);</string>
     </property>
     <property name="text">
      <string>CET4</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="color_ico">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/ui/work.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="date_show">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>290</y>
      <width>271</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: rgb(156, 48, 133);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;版本&lt;/span&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;:0.3&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;制作与发布：计科10班 肖永阳唐润雨&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;图标来源&lt;/span&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;：&lt;/span&gt;&lt;a href=&quot;https://www.iconfont.cn/?spm=a313x.7781069.1998910419.d4d0a486a&quot;&gt;&lt;span style=&quot; font-weight:600; text-decoration: underline; color:#0000ff;&quot;&gt;https://www.iconfont.cn&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;GUI框架&lt;/span&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;：QT5.12&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="margin">
     <number>5</number>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/ui/ditionary_ico.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/ui/qt-logo.ico);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>groupBox_3</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>groupBox</zorder>
   <zorder>set_ico</zorder>
   <zorder>color_ico</zorder>
   <zorder>date_show</zorder>
   <zorder>en_ico</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
  </widget>
  <widget class="QCommandLinkButton" name="me_btn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>47</width>
     <height>52</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="ui.qrc">
     <normaloff>:/ui/my.png</normaloff>:/ui/my.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="set_btn">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>451</y>
     <width>46</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="ui.qrc">
     <normaloff>:/ui/more.png</normaloff>:/ui/more.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="test_btn">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>181</y>
     <width>129</width>
     <height>59</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(157, 38, 111);
font: 75 12pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string> 测试</string>
   </property>
   <property name="icon">
    <iconset resource="ui.qrc">
     <normaloff>:/ui/data.png</normaloff>:/ui/data.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="me_w" native="true">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>571</width>
     <height>461</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <widget class="QLabel" name="my_date">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>271</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;微软雅黑&quot;;
color: rgb(156, 48, 133);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;查找次数：NULL&lt;/p&gt;&lt;p&gt;总分：NULL&lt;/p&gt;&lt;p&gt;测试错误次数：NULL&lt;/p&gt;&lt;p&gt;测试正确次数：NULL&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="my_name">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>0</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>默认用户</string>
    </property>
   </widget>
   <widget class="QLabel" name="my_photo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/ui/ditionary_ico.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="mind" native="true">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>571</width>
     <height>471</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <widget class="QSlider" name="word_number">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>271</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal{ 
		height: 12px; 
		left: 0px; 
		right: 0px; 
		border:0px;    
	    border-radius:6px;    
		background:rgba(0,0,0,50);
 }
QSlider::handle:horizontal{ 
		width:  50px; 
		height: 50px; 
		margin-top: -20px; 
		margin-left: 0px; 
		margin-bottom: -20px; 
		margin-right: 0px; 
border-image:url(:/main_ui/circle.png);
 	
} 

QSlider::sub-page:horizontal{
	   background:rgba(80,166,234,1);
}</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::NoTicks</enum>
    </property>
    <property name="tickInterval">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="mind_show">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>170</y>
      <width>311</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>mind_show</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="word_value">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>90</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>asd</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_btn">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>370</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="ui.qrc">
      <normaloff>:/ui/delete.png</normaloff>:/ui/delete.png</iconset>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>430</y>
      <width>201</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QRadioButton" name="page_mod">
       <property name="styleSheet">
        <string notr="true">font: 75 11pt &quot;微软雅黑&quot;;</string>
       </property>
       <property name="text">
        <string>用户词汇</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="list_mod">
       <property name="styleSheet">
        <string notr="true">font: 75 11pt &quot;微软雅黑&quot;;</string>
       </property>
       <property name="text">
        <string>四级词汇</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QListWidget" name="wordlist">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>551</width>
      <height>391</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>word_value</zorder>
   <zorder>word_number</zorder>
   <zorder>mind_show</zorder>
   <zorder>delete_btn</zorder>
   <zorder>wordlist</zorder>
  </widget>
  <widget class="QCommandLinkButton" name="mind_btn">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>116</y>
     <width>129</width>
     <height>59</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(157, 38, 111);
font: 75 12pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string> 词本</string>
   </property>
   <property name="icon">
    <iconset resource="ui.qrc">
     <normaloff>:/ui/mind_pink.png</normaloff>:/ui/mind_pink.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="add_btn">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>330</y>
     <width>421</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="ui.qrc">
     <normaloff>:/ui/add2.png</normaloff>:/ui/add2.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="word_lab">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>190</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>单词</string>
   </property>
  </widget>
  <zorder>test_btn</zorder>
  <zorder>mind_btn</zorder>
  <zorder>find_btn</zorder>
  <zorder>show_lab</zorder>
  <zorder>word_lab</zorder>
  <zorder>edit_Box</zorder>
  <zorder>add_btn</zorder>
  <zorder>find_edit</zorder>
  <zorder>top</zorder>
  <zorder>explain</zorder>
  <zorder>note</zorder>
  <zorder>explain_lab</zorder>
  <zorder>note_lab</zorder>
  <zorder>set_w</zorder>
  <zorder>me_btn</zorder>
  <zorder>set_btn</zorder>
  <zorder>me_w</zorder>
  <zorder>mind</zorder>
 </widget>
 <resources>
  <include location="ui.qrc"/>
 </resources>
 <connections/>
</ui>
